network/websocket_dataframe.ksy

```diff
@@ -0,0 +1,68 @@
+meta:
+  id: websocket_dataframe
+  title: WebSocket
+  xref:
+    rfc: 6455
+  endian: be
+  license: APACHE-2.0
+doc: |
+  The WebSocket protocol establishes a two-way communication channel via TCP.
+  Messages are made up of one or more dataframes, and are delineated by
+  frames with the `fin` bit set.
+seq:
+  - id: fin
```

Comment:
could you use clearer `id`s, using the names specified in the rfc in `-orig-id`?